lib/tasks/data_hygiene/export_data.rb

```diff
@@ -0,0 +1,44 @@
+module Tasks
+  module DataHygiene
+    class ExportData
+      def initialize(file, stdout)
```

Comment:
I wasn't keen to have a worker object bypassing its wrapper and talking directly to the outside world.  This allows non-error output to be sent to a file within the rake task if we chose to.